docs/legacy-protocols/how-to-authenticate-an-imap-pop-smtp-application-by-using-oauth.md

```diff
@@ -8,21 +8,21 @@ ms.audience: Developer
 
 # Authenticate an IMAP, POP or SMTP connection using OAuth
 
-Learn how to use OAuth authentication to connect with IMAP, POP or SMTP protocols and access email data for Office 365 users.
+Learn how to use OAuth authentication to connect with IMAP, POP, or SMTP protocols and to access email data for Office 365 users.
 
-> OAuth2 support for IMAP, POP, SMTP protocols as described below is supported for both Microsoft 365 (which includes Office on the web) and Outlook.com users.
+> OAuth2 support for IMAP, POP, and SMTP protocols as described below is available  for both Microsoft 365 (which includes Office on the web) and Outlook.com users.
```

Comment:
Why the change from "supported" to "available"? any particular reason?

Reply:
Just felt that the term "support/supported" occurring twice renders the sentence ambiguous; hence, replaced "supported" with "available".

On the basis of your approval of the change during discussion, retaining "available".